apps/api/v2/src/modules/webhooks/outputs/webhook.output.ts

```diff
@@ -0,0 +1,79 @@
+import { ApiProperty } from "@nestjs/swagger";
+import { WebhookTriggerEvents } from "@prisma/client";
+import { Expose, Type } from "class-transformer";
+import { IsBoolean, IsEnum, IsInt, IsOptional, IsString, ValidateNested } from "class-validator";
+
+import { SUCCESS_STATUS, ERROR_STATUS } from "@calcom/platform-constants";
+
+export class WebhookOutputDto {
+  @IsInt()
+  @Expose()
+  readonly id!: number;
+
+  @IsString()
+  @IsOptional()
+  @Expose()
+  readonly appId?: string;
+
+  @IsInt()
+  @Expose()
+  readonly userId?: number;
+
+  @IsInt()
+  @Expose()
+  @IsOptional()
+  readonly teamId?: number;
+
+  @IsInt()
+  @IsOptional()
+  @Expose()
+  readonly eventTypeId?: number;
```

Comment:
I see optional `userId?` and optional `eventTypeId?` and given that this PR. is for webhooks for users and event-types makes me think that we should have different outputs for users and event-types webhooks where we have as little optiona `?` properties as possible.